zerver/openapi/zulip.yaml

```diff
@@ -1491,60 +1491,29 @@ paths:
                                   "id": 28,
                                 }
                             - type: object
-                              additionalProperties: false
```

Comment:
This is how I made the test failure quoted in the PR description go away. I'm not sure if it's what we want.